### PR TITLE
5421 separate api to enable or disable exchange

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: reservesetting/http/exchanges.go [hunk 1]
```diff
@@ -24,10 +25,34 @@ func (s *Server) getExchange(c *gin.Context) {
...
+	if err := c.ShouldBindUri(&input); err != nil {
+		httputil.ResponseFailure(c, httputil.WithError(err))
+		return
+	}
+	exchange, err := s.storage.GetExchange(input.ID)
+	if err != nil {
+		httputil.ResponseFailure(c, httputil.WithError(err))
+		return
+	}
+	newStatus := !exchange.Disable
```
Comment (secmask) [inline reservesetting/http/exchanges.go]: caller should explicit set new status instead of invert current status.

Comment (NNagato) [reply reservesetting/http/exchanges.go]: fixed in https://github.com/KyberNetwork/reserve-data/pull/839/commits/2240d38d546d508538d5c36ef20dc98ea8867d37

**Comment by secmask** [comment]
LGTM

**Comment by secmask** [comment]
@ductm54 cc

**Pull request merged**